config/locales/es/general.yml

```
@@ -933,3 +934,21 @@ es:
   machine_learning:
     comments_summary: "Resumen de comentarios"
     info_text: "Contenido generado mediante IA / Machine Learning"
+  cookies_consent:
+    accept_all_cookies: "Aceptar todo"
+    accept_essential_cookies: "Aceptar cookies esenciales"
+    message: "Las cookies nos ayudan a ofrecer nuestros servicios. Al utilizar nuestros servicios, aceptas el uso de cookies."
+    more_info_link: "Más información sobre cookies"
+    management: "Ajustes"
+    title: "Política de cookies"
+  cookies_management:
+    title: Configuración de cookies
+    description: Esta web utiliza cookies propias y de terceros para su correcto funcionamiento. Al hacer click en el botón Aceptar todo, acepta el uso de estas tecnologías y el procesamiento de tus datos para estos propósitos.
```

Small typo (we're saying "tus datos", so...):

```suggestion
    description: Esta web utiliza cookies propias y de terceros para su correcto funcionamiento. Al hacer click en el botón Aceptar todo, aceptas el uso de estas tecnologías y el procesamiento de tus datos para estos propósitos.
```